lessons/Data-Science/measures-of-center/langs/en-us/pages/histograms-moc.adoc

```diff
@@ -0,0 +1,77 @@
+= Histograms and Measures of Center
+
+@n The two histograms below represent the amount of money spent on a pair of jeans, one for a sample of sixth grade students and one for a sample of eighth grade students.
+
+[cols="^1a,^1a"]
+|===
+
+| @image{../images/jeans-cost-6th.png, 275}  | @image{../images/jeans-cost-8th.png, 275}
+
+|===
+
+Which group has the larger mode? @hspace{1em} *Girls* @hspace{1em} // @hspace{1em} *Boys* @hspace{1em} // @hspace{1em} *The modes are roughly the same.*
+
+
+
+@n The histogram below shows the Verbal SAT scores for 205 students entering a local college in the fall of 2002.
+
+[cols="^1a,<1a"]
+|===
+
+| @image{../images/sat-verbal.png}
+| The median score for these 205 students is:
+
+- about 40 or 41
```

Comment:
@retabak We need checkboxes instead of bullets here. This html character will give you what you need
&#9744;

and for the row that includes the solution you want to use @ifnotsoln{&#9744;}@ifsoln-choice{whatever the line of text is}